release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1105,6 +1105,10 @@ With this release, the CCO no longer checks for the nonexistent permission.
 [id="ocp-4-16-image-registry-bug-fixes_{context}"]
 ==== Image Registry
 
+* Previously, the Image Registry Operator had maintained its own list of {ibm-power-server-name} regions, so any new regions were not added to the list. With this release, the Operator relies on an external library for accessing regions so that it can support new regions. (link:https://issues.redhat.com/browse/OCPBUGS-26767[*OCPBUGS-26767*])
+
+* Previously, the image registry's Azure path fix job incorrectly required the presence of `AZURE_CLIENT_ID` and `TENANT_CLIENT_ID` parameters to function. This caused a valid configuration to throw an error message. With this release, adds a check to the Identity and Access Management (IAM) service account key to validate if these parameters are needed, so that a cluster upgrade operation no longer fails. (link:https://issues.redhat.com/browse/OCPBUGS-32328[*OCPBUGS-32328*]
```

Comment:
```suggestion
* Previously, the image registry's Azure path fix job incorrectly required the presence of `AZURE_CLIENT_ID` and `TENANT_CLIENT_ID` parameters to function. This caused a valid configuration to throw an error message. With this release, a check was added to the Identity and Access Management (IAM) service account key to validate if these parameters are needed, so that a cluster upgrade operation no longer fails. (link:https://issues.redhat.com/browse/OCPBUGS-32328[*OCPBUGS-32328*]
```